src/Microsoft.DotNet.Build.Tasks/Targets/toolruntime.targets

```diff
@@ -17,7 +17,7 @@
     <ToolArchitecture Condition="'$(ToolArchitecture)' == ''">x64</ToolArchitecture>
     <ToolNugetRuntimeId Condition="'$(ToolNugetRuntimeId)' == ''">win7-$(ToolArchitecture)</ToolNugetRuntimeId>
 
-    <ToolRuntimePath Condition="'$(ToolRuntimePath)' == ''">$(BaseOutputPath)$(OSPlatformConfig)\ToolRuntime\</ToolRuntimePath>
+    <ToolRuntimePath Condition="'$(ToolRuntimePath)' == ''">$(IntermediateOutputPath)ToolRuntime\</ToolRuntimePath>
```

Comment:
This should be $(IntermediateOutputRootPath) because we want it to be shared with all the projects and $(IntermediateOutputPath) includes the project name in the path. See https://github.com/dotnet/corefx/blob/master/dir.props#L226